velox/connectors/hive/HiveConfig.h

```diff
@@ -240,6 +240,54 @@ class HiveConfig {
   static constexpr const char* kCacheNoRetention = "cache.no_retention";
   static constexpr const char* kCacheNoRetentionSession = "cache.no_retention";
 
+  const std::unordered_set<std::string> kHiveConfigs = {
```

Comment:
I feel Prestissimo should handle this.

Reply:
Thanks for the suggestion, moved the changes to `PrestoServer.cpp`, could you please take a look: https://github.com/prestodb/presto/pull/23449/